homework/src/nyc/c4q/ahhhlvin/DomesticCat.java

```diff
@@ -0,0 +1,43 @@
+package nyc.c4q.ahhhlvin;
+/**
+ * Created by alvin2 on 4/29/15.
+ * Alvin Kuang
+ * C4Q Access Code 2.1
+ */
+public class DomesticCat extends Animal implements Domestic
+{
+    String name;
+
+    public DomesticCat(String speciesName, String name)
+    {
+        super(speciesName);
+        this.name = name;
+    }
+
+    @Override
```

Comment:
**work in progress...will come back to this**

Not necessary to `@Override` since you're not technically overriding...you are just implementing a method you promised to have